examples/visualization/plot_clickable_image.py

```diff
@@ -56,7 +56,7 @@
 sr = 1000
 nsec = 5
 events = np.arange(nepochs).reshape([-1, 1])
-events = np.hstack([events, np.zeros([nepochs, 2])])
+events = np.hstack([events, np.zeros([nepochs, 2])]).astype(int)
```

Comment:
events = np.hstack([events, np.zeros([nepochs, 2], dtype=int)])

is cleaner. please update and back port.